packages/cloud_functions/CHANGELOG.md

```diff
@@ -1,95 +1,99 @@
+## 0.4.0+3
```

Comment:
Hello and thanks for the suggestions. I would appreciate and help me understand when the build number should be increased and when not for flutter projects.

Reply:
We're using semVer, so when a new API is added you should increment the minor revision. Since this a pre-1.0 plugin that involves the last number before the plus.